src/main/scala/org/camunda/feel/impl/parser/FeelParser.scala

```diff
@@ -423,9 +423,13 @@ object FeelParser {
     ).map(entries => ConstContext(entries.toList))
 
   private def contextEntry[_: P]: P[(String, Exp)] = P(
-    (name | stringWithQuotes) ~ ":" ~ expression
+    (contextAnySymbol | identifierWithWhitespaces | name | stringWithQuotes) ~ ":" ~ expression
   )
 
+  private def contextAnySymbol[_: P]: P[String] = {
+    P((!"\"" ~ !"{" ~ !"}" ~ !":" ~ !"," ~ AnyChar ~ !"{").rep(1).!)
+  }
```

Comment:
:wrench: We could extract the reserved symbols in a method. This would make it a bit easier to read.

```
private def reservedSymbol[_: P]: P[String] = P(
    CharIn("\"" , "{" , "}" , ":" , ",", "[", "]", "`")
  ).!

private def contextKeyAnySymbol[_: P]: P[String] = P(
    (!reservedSymbol ~ AnyChar).rep(1)
  ).!
```

Reply:
Cool, it looks way better!

Reply:
Method added. As a matter of clean code principles, I swapped the order of methods, namely:

```
private def contextKeyAnySymbol[_: P]: P[String] = P(
    (!reservedSymbol ~ AnyChar).rep(1)
  ).!
  
  private def reservedSymbol[_: P]: P[String] = P(
    CharIn("\"" , "{" , "}" , ":" , ",", "[", "]", "`")
  ).!
```